thirdeye/thirdeye-frontend/app/pods/components/rootcause-metrics/component.js

```diff
@@ -65,10 +55,29 @@ export default Component.extend({
    */
   onSelection: null, // function (Set, state)
 
+  context: null,
+
+  compareMode: null, // ""
+
+  didReceiveAttrs() {
+    const context = get(this, 'context');
+
+    set(this, 'compareMode', (context || {}).compareMode);
+  },
+
   //
   // internal properties
   //
 
+  /**
+   * Columns for metrics table
+   * @type Object[]
+   */
+  metricsTableColumns: computed('compareMode', function () {
+    const compareMode = get(this, 'compareMode');
+    return compareMode === 'forecast' ? METRIC_FUNNEL_COLUMNS : METRICS_TABLE_COLUMNS;
```

Comment:
So the funnel table shows up when the forecast is selected, correct?

Reply:
Yes, but only if the user clicks on the Metrics tab.  Do we want it to automatically select the Metrics table when the user selects forecast?